src/libclipper/src/metrics.cpp

```diff
@@ -1,3 +1,4 @@
+#include <algorithm>
```

Comment:
`std::sort` which we use in this file is defined in `<algorithm>`. I prefer to explicitly include the header dependencies for a file as much as possible and I noticed this one was missing.